understanding/22/focus-appearance-minimum.html

```diff
@@ -22,16 +22,17 @@ <h2>Focus Appearance (Minimum) Success Criteria text</h2>
             <ul>
-                <li><strong>Minimum area:</strong> The area of the <a>focus indicator</a> is at least as large as as the area of a 1 <a>CSS pixel</a> thick <a>perimeter</a> of the focused control, or the focus indicator has a thickness of at least 8 CSS pixels along the shortest side of the user interface component.</li>
-                <li><strong>Change of contrast:</strong> The pixels making up the minimum focus indicator area achieve at least a 3:1 contrast between their colors in the focused and unfocused states. </li>
-                <li><strong>Adjacent contrast:</strong> The pixels in the minimum focus indicator area achieve at least a 3:1 against adjacent colors in the focused control or the minimum focus indicator area has a thickness of at least 2 CSS pixels.</li>
-                <li><strong>Not fully obscured:</strong> The item with focus is not entirely hidden by author-created content.</li>
+              <li><strong>Minimum area:</strong> The area of the <a>focus indicator</a> is at least as large as the area of a 1 <a>CSS pixel</a> thick <a>perimeter</a> of the focused control, or the focus indicator has a thickness of at least 8 CSS pixels along the shortest side of the user interface component.</li>      
+              <li><strong>Change of contrast:</strong> The pixels making up the minimum focus indicator area achieve at least a 3:1 contrast between their colors in the focused and unfocused states.</li>
+              <li><strong>Adjacent contrast:</strong> The pixels in the minimum focus indicator area achieve at least a 3:1 against adjacent colors in the focused control or the minimum focus indicator area has a thickness of at least 2 CSS pixels.</li>
+              <li><strong>Not fully obscured:</strong> The item with focus is not entirely hidden by author-created content.</li>
             </ul>
             <p class="note">A keyboard focus indicator which has a pattern or gradient may have parts that do not meet the 3:1 contrast ratio for the change of contrast, as long as an area equal to the minimum does meet the contrast ratio.</p>
             <p class="note">If the control has a visible boundary smaller than the hit area, the size measure is taken from the visible boundary.</p>
-            <dt class="new"><dfn>Focus indication area</dfn></dt>
+            <p class="note">The default user agent focus indicator is not excepted from the requirements of Focus Appearance (Minimum). If a user agent implementation fails to meet these requirements, any page relying on that user agent would fail.</p>
+            <dt class="new"><dfn>Focus indicator</dfn></dt>
             <dd class="new">
-	            <p>the pixels that change between the focused and unfocused states of a <a>User Interface Component</a></p>   				
+	            <p>The pixels that are changed to visually indicate when a user interface component is in a focused state.</p>   				
```

Comment:
Again, for consistency:

 <p>pixels that are changed to visually indicate when a user interface component is in a focused state</p>